imagersite/imagersite/tests.py

```diff
@@ -0,0 +1,354 @@
+            file=os.path.join(settings.MEDIA_ROOT, 'amoosing.jpg'))
+        self.photo2 = PhotoFactory.create(owner=self.user, privacy='Public',
+            file=os.path.join(settings.MEDIA_ROOT, 'googlephoto.jpg'))
+        self.photo1.save()
+        self.photo2.save()
+        self.response1 = self.client.get('/images/photos/' +
+                                    str(self.photo1.id) +'/')
+        self.response2 = self.client.get('/images/photos/' +
+                                    str(self.photo2.id) +'/')
+
+    def test_check_photo_url_exists(self):
+        self.assertEqual(self.response1.status_code, 200)
+        self.assertEqual(self.response2.status_code, 200)
+
+    def test_photo_displayed(self):
+        # Getting the html tag content for thumbnail
+        atemp = Template('src="{{ MEDIA_URL }}{{ photo.file }}"')
+        srclink1 = atemp.render(Context({'MEDIA_URL': settings.MEDIA_URL,
+            'photo': self.photo1}))
+        srclink2 = atemp.render(Context({'MEDIA_URL': settings.MEDIA_URL,
+            'photo': self.photo2}))
+        self.assertIn(srclink1, self.response1.content)
+        self.assertIn(srclink2, self.response2.content)
+
+
+class AlbumView(TestCase):
+    """Tests for Album view"""
+    def setUp(self):
+        self.client = Client()
+        # Fake data
+        self.username = fake.user_name()
+        self.password = fake.password()
+        self.email = fake.email()
+        # Create user
+        self.user = User.objects.create_user(username=self.username,
+            password=self.password, email=self.email)
+        self.login = self.client.post('/login/', {'username': self.username,
+            'password': self.password}, follow=True)
+        # Make photos
+        self.photo1 = PhotoFactory.create(owner=self.user, privacy='Public',
+            file=os.path.join(settings.MEDIA_ROOT, 'amoosing.jpg'))
+        self.photo2 = PhotoFactory.create(owner=self.user, privacy='Public',
+            file=os.path.join(settings.MEDIA_ROOT, 'googlephoto.jpg'))
+        self.photo1.save()
+        self.photo2.save()
+        self.album = AlbumFactory.create(owner=self.user, cover=self.photo1)
+        self.album.save()
+        self.album.photos = [self.photo1, self.photo2]
+        self.response = self.client.get('/images/albums/' +
+                                        str(self.album.id) + '/')
+
+    def test_photo_view_url_exists(self):
+        self.assertEqual(self.response.status_code, 200)
+
+    def test_photos_displayed(self):
+        # Getting the html tag content for thumbnails
+        atemp = Template('src="{{ MEDIA_URL }}{{ photo.file }}"')
+        srclink1 = atemp.render(Context({'MEDIA_URL': settings.MEDIA_URL,
+            'photo': self.photo1}))
+        srclink2 = atemp.render(Context({'MEDIA_URL': settings.MEDIA_URL,
+            'photo': self.photo2}))
+        self.assertIn(srclink1, self.response.content)
+        self.assertIn(srclink2, self.response.content)
```

Comment:
I can agree that the Home page view tests belong here, but the profile view tests probably belong in the profile app, and the views for albums and photos in the images app.